### ISSUE TITLE
1.4.x.x version not run on latest stable Android Studio (3.3.2)

### ISSUE THREAD

**Issue by rooton** [mannodermaus/android-junit5#164]
Gradle - 5.3.1

```
dependencies {
        classpath "de.mannodermaus.gradle.plugins:android-junit5:1.4.2.0"
}
```

**Throw:**

This version of Android Studio cannot open this project, please retry with Android Studio 3.4 or newer.

**Comment by nzakharchenko** [comment]
For me, only version `1.4.0.0` works

**Comment by Jendrekpl** [comment]
For me even 1.4.0.0 after upgrade android studio to 3.5 canary 12 is not working.

ERROR: No such property: javaOutputDir for class: com.android.build.gradle.internal.scope.VariantScopeImpl

**Comment by mannodermaus** [comment]
The android-junit5 plugin has a transitive dependency on the Android Gradle Plugin, and causes it to be bumped to whatever its internal version is. There is backwards compatibility back to AGP 3.2.x, but the latest version accidentally bumps this up to 3.4.0, causing this problem in AS 3.3.

To get around this, please use the now-stable AS 3.4. If you can't, here is a workaround: If you exclude the transitive dependency on the AGP in your build.gradle file, it won't be overwritten:

```groovy
buildscript {
    dependencies {
        classpath("de.mannodermaus.gradle.plugins:android-junit5:1.4.2.0") {
            exclude group: "com.android.tools.build", module: "gradle"
        }
    }
}
```

@Jendrekpl, this is unrelated to this ticket at hand, and sounds more like #167.

**Comment by mannodermaus** [comment]
This should be resolved in the latest `1.4.2.1-SNAPSHOT`, and scheduled for proper release in the next version.

**Comment by nzakharchenko** [comment]
Thx a lot for taking care! 👍
